html-forms/index.html

```diff
@@ -23,29 +23,34 @@ <h1>Coding Form!</h1>
         </select>
       </div>
     </label>
-    <label for="languages">
+    <span>
       Favorite Language (choose one)
-      <div>
-        <input type="radio" name="language" value="html" /> HTML
-        <input type="radio" name="language" value="css" /> CSS
-        <input type="radio" name="language" value="javascript" /> JavaScript
-      </div>
-    </label>
-    <label for="information">
+    </span>
+    <div>
+      <input type="radio" name="language" value="html" />
+      <label for="HTML">HTML</label>
+      <input type="radio" name="language" value="css" />
+      <label for="CSS">CSS</label>
+      <input type="radio" name="language" value="javascript" />
+      <label for="JavaScript">JavaScript</label>
```

Comment:
These `input` elements need an `id` attribute that matches the `for` attribute on the label